boards.txt

```diff
@@ -5716,6 +5716,362 @@ hornbill32minima.menu.DebugLevel.debug.build.code_debug=4
+dfrobot_beetle_esp32c3.menu.PartitionScheme.minimal.build.partitions=minimal
+dfrobot_beetle_esp32c3.menu.PartitionScheme.no_ota=No OTA (2MB APP/2MB SPIFFS)
+dfrobot_beetle_esp32c3.menu.PartitionScheme.no_ota.build.partitions=no_ota
+dfrobot_beetle_esp32c3.menu.PartitionScheme.no_ota.upload.maximum_size=2097152
+dfrobot_beetle_esp32c3.menu.PartitionScheme.noota_3g=No OTA (1MB APP/3MB SPIFFS)
+dfrobot_beetle_esp32c3.menu.PartitionScheme.noota_3g.build.partitions=noota_3g
+dfrobot_beetle_esp32c3.menu.PartitionScheme.noota_3g.upload.maximum_size=1048576
+dfrobot_beetle_esp32c3.menu.PartitionScheme.noota_ffat=No OTA (2MB APP/2MB FATFS)
+dfrobot_beetle_esp32c3.menu.PartitionScheme.noota_ffat.build.partitions=noota_ffat
+dfrobot_beetle_esp32c3.menu.PartitionScheme.noota_ffat.upload.maximum_size=2097152
+dfrobot_beetle_esp32c3.menu.PartitionScheme.noota_3gffat=No OTA (1MB APP/3MB FATFS)
+dfrobot_beetle_esp32c3.menu.PartitionScheme.noota_3gffat.build.partitions=noota_3gffat
+dfrobot_beetle_esp32c3.menu.PartitionScheme.noota_3gffat.upload.maximum_size=1048576
+dfrobot_beetle_esp32c3.menu.PartitionScheme.huge_app=Huge APP (3MB No OTA/1MB SPIFFS)
+dfrobot_beetle_esp32c3.menu.PartitionScheme.huge_app.build.partitions=huge_app
+dfrobot_beetle_esp32c3.menu.PartitionScheme.huge_app.upload.maximum_size=3145728
+dfrobot_beetle_esp32c3.menu.PartitionScheme.min_spiffs=Minimal SPIFFS (1.9MB APP with OTA/190KB SPIFFS)
+dfrobot_beetle_esp32c3.menu.PartitionScheme.min_spiffs.build.partitions=min_spiffs
+dfrobot_beetle_esp32c3.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
+dfrobot_beetle_esp32c3.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FAT)
+dfrobot_beetle_esp32c3.menu.PartitionScheme.fatflash.build.partitions=ffat
+dfrobot_beetle_esp32c3.menu.PartitionScheme.fatflash.upload.maximum_size=2097152
+dfrobot_beetle_esp32c3.menu.PartitionScheme.app3M_fat9M_16MB=16M Flash (3MB APP/9MB FATFS)
+dfrobot_beetle_esp32c3.menu.PartitionScheme.app3M_fat9M_16MB.build.partitions=app3M_fat9M_16MB
+dfrobot_beetle_esp32c3.menu.PartitionScheme.app3M_fat9M_16MB.upload.maximum_size=3145728
+dfrobot_beetle_esp32c3.menu.PartitionScheme.rainmaker=RainMaker
+dfrobot_beetle_esp32c3.menu.PartitionScheme.rainmaker.build.partitions=rainmaker
+dfrobot_beetle_esp32c3.menu.PartitionScheme.rainmaker.upload.maximum_size=3145728
+
+dfrobot_beetle_esp32c3.menu.CPUFreq.160=160MHz (WiFi)
+dfrobot_beetle_esp32c3.menu.CPUFreq.160.build.f_cpu=160000000L
+dfrobot_beetle_esp32c3.menu.CPUFreq.80=80MHz (WiFi)
+dfrobot_beetle_esp32c3.menu.CPUFreq.80.build.f_cpu=80000000L
+dfrobot_beetle_esp32c3.menu.CPUFreq.40=40MHz
+dfrobot_beetle_esp32c3.menu.CPUFreq.40.build.f_cpu=40000000L
+dfrobot_beetle_esp32c3.menu.CPUFreq.20=20MHz
+dfrobot_beetle_esp32c3.menu.CPUFreq.20.build.f_cpu=20000000L
+dfrobot_beetle_esp32c3.menu.CPUFreq.10=10MHz
+dfrobot_beetle_esp32c3.menu.CPUFreq.10.build.f_cpu=10000000L
+
+dfrobot_beetle_esp32c3.menu.FlashMode.qio=QIO
```

Comment:
same question about flash options

Reply:
The flash selection is reserved, and more modules may be added in the future